js/_api.js

```diff
@@ -20,21 +20,27 @@ export function call(uri, method, payload) {
 		credentials: 'same-origin',
 		method: method,
 		headers: {
-			Accept: 'application/json',
-			'Content-Type': 'application/json'
+			Accept: 'application/json'
 		}
 	};
 
-	if(payload) {
-		var data = new FormData();
-		data.append("json", JSON.stringify(payload));
-		options.body = JSON.stringify(payload);
+	if(method.toLowerCase() === "post") {
```

Comment:
I regard all post calls to send the csrf token back, since they should by some standards of http APIs create/update/delete resources. I currently don't see any need to send csrf tokens on GET calls.